src/jaxsim/rbda/crba.py

```diff
@@ -128,10 +132,21 @@ def compute_inner(carry: CarryInnerFn) -> tuple[CarryInnerFn, None]:
                 operand=carry,
             )
 
-        (j, Fi, M), _ = jax.lax.scan(
-            f=inner_fn,
-            init=carry_inner_fn,
-            xs=jnp.flip(jnp.arange(start=1, stop=model.number_of_links())),
+        (j, Fi, M), _ = (
+            jax.lax.scan(
+                f=inner_fn,
+                init=carry_inner_fn,
+                xs=jnp.flip(jnp.arange(start=1, stop=model.number_of_links())),
+            )
+            if model.number_of_links() > 1
+            else [
+                (
+                    j,
+                    Fi,
+                    M,
```

Comment:
Can you remove the comma after `M`? Maybe black compacts this tuple better.

Reply:
Addressed in https://github.com/ami-iit/jaxsim/pull/219/commits/aa87ee417f6ea1d6ca6e036a554383bf75c5511f, thank you!